crates/forge/src/cmd/compiler.rs

```diff
@@ -59,18 +66,17 @@ impl ResolveArgs {
         let Self { root, skip } = self;
 
         let root = root.unwrap_or_else(|| PathBuf::from("."));
-        let config = Config::load_with_root(&root)?;
+        let config = Config::load_with_root(&root)?.canonic_at(root);
```

Comment:
why is it needed? How it worked before?

Reply:
that's the fun part, for me at least it didn't when passing `--root ../<another sibling dir>` because it would look for sources inside `./src` instead of `../<another sibling dir>/src`